{% extends "layout.html" %}

{% block page_content %}

<style>

@font-face
    {
        font-family:'font_one';
        src:url(https://static.igem.wiki/teams/4242/wiki/acme-regular.ttf)
    }
    .p1
    {
        font-family:'font_one';
        font-size:1.5rem;
        color:#f0f0f0;
    }
    
    @font-face
    {
        font-family:'font_two';
        src:url(https://uploads.igem.org/teams/4242/wiki/Overlock-Regular.ttf)

    }
    .o
    {
        font-family:'font_two';
        font-size:1.2rem;
        color:#f0f0f0;
    }
    .z1
    {
        font-family:'font_one';
        font-size:1.5rem;
        color:#f26b1f
    }
    .z2
    {
        font-family:'font_two';
        font-size:0.7rem;
        color:#f0f0f0
    }
    
    body
   {
        width:100%;
        height: auto;
        background-color: #21373d;
        overflow-y: scroll;
    }
    #move20
    {
      margin-left: 20%;
    }

    @media screen and (max-width: 992px) 
    {
      #move20 {
      margin-left: 0%;
    }
    }
</style>


<head>
    <meta charset="UTF-8" />
    <title>Team:CUG-China/Notebook - 2022.igem.org</title>
    <meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0,user-scalable=no">
    <link rel="stylesheet" href="{{ url_for('static', filename = 'headama.css') }}">
    <link rel="stylesheet" href="{{ url_for('static', filename = 'notebook amazing/nb.css') }}">
</head>

<body>
    <section class="section main-banner" id="top" data-section="Intro">
        <img src="https://static.igem.wiki/teams/4242/wiki/background/background/b-protocols.png" alt="">
        <div class="header-text">
            <div class="caption">
                <div class="wave-text">
                    <span>Engineering</span>
                </div>
            </div>
        </div>
    </section>

    <div class="container">
          {% include "sider.html" %}
          <aside class="aside">
            <br>
            <div>
              <a class="pointer" href="#Overview">Overview</a>
              <a class="pointer" href="#Cycle 1">Cycle 1</a>
              <a class="pointer" href="#Cycle 2">Cycle 2</a>
              <a class="pointer" href="#Cycle 3">Cycle 3</a>
            </div>
          </br>
          </aside>
      
        <div class="row" id="move20">     
          <div class="col" id="Content">
          <!---Overview--->
          <p class="target-fix" id="Overview"></p>
          <p style="margin-top: 50px;"></p>
          <div class="p1">
         <h1 id="Overview;" style="color: #f26b1f;"><b>Overview</b></h1> 
         </div> 
         <div class="o">
         <hr style="border-top:5px solid #f0f0f0;">
         <ul>
            <p>In order to connect the c-di-GMP level to the reporter gene, we chose the signaling system FleQ protein from <i>P. aeruginosa</i> in which the FleQ protein functions as both a repressor and an activator to control gene expression from the <i>pel</i> operon promoter in response to c-di-GMP. However, how to make this system work in other bacteria is a big challenge. We have experienced three engineering cycles to construct a preliminary functional biosensor.</p>
         </ul>
         </hr>
         </div>
        
        <!---Cycle 1--->
        <p class="target-fix" id="Cycle 1"></p>
        <p style="margin-top: 50px;"></p>
        <div class="p1">
       <h1 id="Cycle 1;" style="color: #f26b1f;"><b>Cycle 1: Test the performance of promotor P<span class="z1">pel</span> in <i>E. coli</i>  and <i>S. oneidensis</i></b></h1> 
       </div> 
       <div class="o">
       <hr style="border-top:5px solid #f0f0f0;">
       <ul>
          <p>Theoretically, the gene under <i>pel</i> promoter can be constitutively expressed in the bacteria that have no repressor FleQ. To test this hypothesis, we directly fused <i>pel</i> promoter to <i>gfp</i> and we had our first plasmid, pHYD-1. Then we transferred the biobrick into two module organisms, <i>Escherichia coli</i> and <i>Shewanella oneidensis</i>. However, no detectable fluorescence was observed, which suggests the promoter P<span class="z2">pel</span> is non-functional in these two bacteria.</p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/wet-pho/6.png" alt="Pulpit rock" width="630" height="230"> 
           </p>
           <p style="text-align: center"><span class="z2">Fig. 1  a|Structure of pHYD-1 and control plasmid  b|Colony RPC result of pHYD-1</span></p>


       </ul>
       </hr>
       </div>

       <!---Cycle 2--->
       <p class="target-fix" id="Cycle 2"></p>
       <p style="margin-top: 50px;"></p>
       <div class="p1">
      <h1 id="Cycle 2;" style="color: #f26b1f;"><b>Cycle 2: Construct the tandem promotor</b></h1> 
      </div> 
      <div class="o">
      <hr style="border-top:5px solid #f0f0f0">
      <ul>
        <p>To overcome the difficulty we met in cycle1, we designed a tandem promoter P<sub>cI/pel</sub> in which the transcription start site of the constitutive promoter P<span class="z2">cI</span> is immediately followed by the FleQ binding site of the promoter P<span class="z2">pel</span>. Transcription of the reporter gene <i>gfp</i> is promoted by <span class="z2">cI</span> promoter in the absence of FleQ. We found that the fluorescence of recombinant <i>E. coli</i>/pHYD-2 and <i>S. oneidensis</i>/pHYD-2 was drastically higher than that of the strains harboring pHYD-1 (Fig. 4) , indicating that the tandem promoter is capable of constitutively promoting <i>gfp</i> expression in the absence of FleQ in both strains.</p>
        <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/wet-pho/7.png" alt="Pulpit rock" width="630" height="230"> 
           </p>
           <p style="text-align: center"><span class="z2">Fig. 2  a|Structure of pHYD-2  b|Colony RPC result of pHYD-2</span></p> 
      </ul>
      </hr>
      </div>

      <!---Cycle 3--->
      <p class="target-fix" id="Cycle 3"></p>
      <p style="margin-top: 50px;"></p>
      <div class="p1">
     <h1 id="Cycle 3;" style="color: #f26b1f"><b>Cycle 3: Add the existence of FleQ</b></h1> 
     </div> 
     <div class="o">
     <hr style="border-top:5px solid #f0f0f0;">
     <ul>
        <p>To test whether FleQ can repress <i>gfp</i> expression under the control of the tandem promoter, we cloned <i>fleQ</i> gene from the genome of <i>P. aeruginosa</i> and inserted it into pHYD-2, generating the plasmid pHYD-3 that contains the complete c-di-GMP biosensor. FleQ represses the transcription of <i>gfp</i> though inhibition of the RNA polymerase to access DNA by binding to P<span class="z2">pel</span> when the level of c-di-GMP is low. At high intracellular c-di-GMP, FleQ undergoes a conformational change when it binds c-di-GMP, which relieves FleQ from <i>pel</i> promoter and leads to <i>gfp</i> gene expression under the control of the tandem promoter. As FleQ represses the expression of <i>gfp</i> by binding to the promoter P<span class="z2">pel</span>, the fluorescence of <i>E. coli</i>/pHYD-3 and <i>S. oneidensis</i>/pHYD-3 expressing the repressor FleQ (Fig. 4) was significantly decreased compared to that of the strains with pHYD-2 that has no FleQ. Taken together, the results showed the tandem promoter is effective in promoting expression of the reporter gene(<i>gfp</i>), and the transcription factor FleQ is able to decrease <i>gfp</i> expression level. <p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/wet-pho/8.png" alt="Pulpit rock" width="630" height="230"> 
               </p>
               <p style="text-align: center"><span class="z2">Fig.3 a|Structure of pHYD-3 b| Colony RPC result of pHYD-3</span></p> 
        <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/wet-pho/9.png" alt="Pulpit rock" width="575" height="462"> 
                </p>
                <p style="text-align: center"><span class="z2">Fig.4 a|engineering vectors b| Normalized fluorescence intensity in <i>E.coli</i> BL21 
                    c| Normalized fluorescence intensity in <i>S. oneidensis</i> MR-1</span></p> 
        
    </ul>
     </hr>
     </div>

     
      </hr>
      </div> 
      </div>
    </div>
<body>

{% endblock %}
