{% extends "layout.html" %}
  
{% block title %}Improvement of an Existing Part{% endblock %}
{% block lead %}Describe the improvements your team made of an existing part.{% endblock %}

{% block page_content %}

<!-- <div class="row mt-4">
  <div class="col">
    <div class="bd-callout bd-callout-info">
      <h4>Gold Medal Criterion #2</h4>
      <p>Make a new Part that improves the function of an existing Part. This improvement must be distinct from your work for Bronze and Silver medals.</p>
      <hr>
      <p>Please see the <a href="https://competition.igem.org/judging/medals">2022 Medals Page</a> for more information.</p>
    </div>
    
    <div class="bd-callout bd-callout-warning">
      <h4>Note</h4>
      <p>You must document your improvement on both the existing and new Parts' Main Pages on the <a href="http://parts.igem.org">Registry</a> for your team to be eligible for this criteria.</p>
    </div>
  </div>
</div> -->
<style>
/* Navagation Bar */
/* Works only in media wiki */
#bodyContent nav a[href ^="https://"]:hover, .link-https:hover{
    background:#afcee2;
    padding-right:none;
}
/*fix public engagement oversize*/
#bodyContent nav a[href ^="https://"], .link-https {
    padding-right:1.5rem;
}
/* compansate the background above */
#bodyContent .navbar .navbar-brand:hover{
    background:none;
}
a:visited {
    color:black;
}
.fixed-top {
    top: 16px;
}
nav {
    background-color: #172544;
    opacity:0.96;
    font-size:1.3rem;
}
.navbar {
    padding: 0rem 1rem;
}
nav .fake {
    width:56px;
    height:40px;
}
.navbar-brand {
    width: 30vw;
    height:auto;
    margin-right: 0;
}

.navbar-brand img {
    width: 100%;
    height: auto;
}

.navbar-dark .navbar-nav .nav-link {
    color:white;
}
.dropdown-menu {
    background-color: #fffdf6;
    border-radius: 0 0 .25rem .25rem;
    margin-top:0;
    border-style: none;
    display:none;
    z-index:5;
}
.dropdown:hover .dropdown-menu {
    display: block;
}
.dropdown-menu::before {
    /*在menu跟dropdown中間加一塊透明的block*/
    content: "";
    display: block;
    position: absolute;
    height: 100px;
    width:160px;
    top: -30px;
    left: 0;
    background-color:transparent;
    cursor: pointer;
    z-index:-2;
}

.dropdown > .dropdown-toggle:active {
    /*Without this, clicking will make it sticky*/
    /*target active才不會讓滑鼠滑上去的時候沒改變*/
    pointer-events: none;
}

/* Large devices (desktops, 992px and up)*/
@media (min-width: 992px) { 
    .navbar-expand-lg .navbar-nav.navbar-nav {
        flex-grow: 1;
        padding: 0 10% 0 15%;
    }
    .navbar-brand {
        width: 13vw;
        height:auto;
    }
}
.bread {
    height: 2.5rem;
    margin: 0;
    padding: 0;
    position: relative;
    opacity:0.96;
    z-index:-5;
}
.bread .shape {
    width: calc(20% - 2.5rem);
    height: 100%;
    display: inline-flex;
    align-items: center;
    justify-content: center;
    position: absolute;
    background: #172544;
}
.bread::after {
    content: ""; 
    position: absolute;
    left: calc(20% - 2.5rem);
    width: 0;
    height: 0;
    border-top: 2.5rem solid #172544;
    border-right: 2.5rem solid transparent;
}
#breadtext{
  font-style:none;
  color:white;
  margin: 0;
  font-size: 0.9rem;
}
#breadtext a{
  color: #A6A7A7;
}
#breadtext a:link, #breadtext a:visited{
  color: white;
}
#navbarDropdownMenuLink span{
    font-size: 1.3rem;
}
</style>
<style>
@media (max-width:576px){
    main table{
        font-size: 0.6rem;
    }
}
</style>
<body data-spy="scroll" data-target=".sidenav" data-offset="101">
    <div class="content container-fluid">
        <div class="row">
            <!--Sidebar (Remember to change the subtitle context, but don't change the id)-->
                <aside class="sidenav d-none d-lg-flex flex-column justify-content-start align-items-center col-2">
                    <ul class="nav flex-column">
                        <li class="nav-item">
                            <a class="nav-link" href="#Subtitle1">Overview</a>
                        </li>
                        <li class="nav-item">
                            <a class="nav-link" href="#Subtitle2">Experimental Results</a>
                        </li>
                     </ul>
                </aside>
            <main class="">
            <!--The Page Name-->
                <div class="pageName">
                    <h1>Improvement</h1>
                    <div class="divider"></div>
                </div>
                <section>
            <!--section name:Overview-->
                    <h2 id ="Subtitle1">Overview</h2>
                        <p>    In our project, one of the major goals is to reduce the uremic toxin, <i>p</i>-Cresol. <i>Clostridium</i> bacteria inside the gut will ferment excess tyrosine into <i>p</i>-Cresol. With the help of <a href="https://2019.igem.org/Team:NCKU_Tainan/Parts" target="_blank">tyrosine ammonia-lyase (TAL)</a> , we are able to convert this excess tyrosine into a harmless product, <i>p</i>-Coumaric acid instead of the toxic <i>p</i>-Cresol.  As such, the concentration of <i>p</i>-Cresol can be reduced.</p>
                        <p>    For our construct design, we focused on <a href="http://parts.igem.org/Part:BBa_I742148" target="_blank">BioBrick BBa_I742148</a> from the <a href="https://2007.igem.org/Edinburgh" target="_blank">2007 iGEM Edinburgh team</a><sup>[<a href="#ref1">1</a>]</sup> in the iGEM BioBrick library. This enzyme has been shown to have the highest Km value compared to other TAL enzyme. </p>
                        <p>    We hypothesized that the native ribosome binding site (NRBS) from <i>Saccharothrix espanaensis</i> will not have a high translation efficiency in <i>E. coli</i> Nissle, so we changed the NRBS in front of TAL into a strong ribosome binding site B0034. Also, we noticed that there was no spacer sequence between NRBS and the TAL coding sequence(CDS) in <a href="http://parts.igem.org/Part:BBa_I742148" target="_blank">BBa_I742148</a>. </p>
                        <p>    According to research, the length of the spacer sequence between RBS and CDS strongly affects translation efficiency.<sup>[<a href="#ref2">2</a>]</sup>  We decided to follow the standard iGEM BioBrick assembly rule to add 6 bp of scar sequence between RBS and CDS. The schematic below depicts how we improved the BioBrick.</p>
                        <div class="container-fluid p-0">
                            <div class="row no-gutters">
                                <div class="col-lg ">
                                    <figure class="d-flex flex-column justify-content-center align-items-center px-lg-3">
                                        <!-- put link in href &amp; src , width in a can adjust-->
                                        <a href="https://static.igem.org/mediawiki/2019/a/a9/T--NCKU_Tainan--Improve_improvement_biobrick.png" target="_blank" style="width:50%"><img src="https://static.igem.org/mediawiki/2019/a/a9/T--NCKU_Tainan--Improve_improvement_biobrick.png" alt="" title="" style="width:100%"></a>
                                        <figcaption>Fig. 1. A schematic depicting our approach.</figcaption>
                                    </figure>
                                </div>
                            </div>
                        </div>
                </section>
                <hr>
                <section>
            <!--section name:Experimental Approach &amp; Results-->
                    <h2 id ="Subtitle2">Experimental Results</h2>
                        <p>    We wanted to prove that TAL expression was improved with the change of RBS (<a href="http://parts.igem.org/Part:BBa_I742146" target="_blank">BBa_I742146</a> into B0034). First, RT-PCR experiment was used to confirm that the constructed TAL Biobrick is being transcribed. As seen in Fig.2, cDNA for both bacteria carrying TAL constructs are being detected by PCR, thus confirming that the TAL genes is actually being transcribed in <i>E. coli</i> Nissle.</p>
                        <div class="container-fluid p-0">
                            <div class="row no-gutters">
                                <div class="col-lg ">
                                    <figure class="d-flex flex-column justify-content-center align-items-center px-lg-3">
                                        <!-- put link in href &amp; src , width in a can adjust-->
                                        <a href="https://static.igem.org/mediawiki/2019/6/62/T--NCKU_Tainan--RT-PCR.png" target="_blank" style="width:70%"><img src="https://static.igem.org/mediawiki/2019/6/62/T--NCKU_Tainan--RT-PCR.png" alt="" title="" style="width:100%"></a>
                                    </figure>
                                    <div class="table-responsive" style="overflow-x:auto;">
                                        <table class="table-bordered table-striped" style="word-wrap:break-word;">
                                            <thead>
                                                <tr>
                                                    <th style="text-align: center">Lane</th>
                                                    <th style="text-align: center">Template</th>
                                                    <th style="text-align: center">Primers</th>
                                                    <th style="text-align: center">Lane</th>
                                                    <th style="text-align: center">Template</th>
                                                    <th style="text-align: center">Primers</th>
                                                </tr>
                                            </thead>
                                            <tbody>
                                                <tr>
                                                    <td style="text-align: center">1</td>
                                                    <td>PCR positive control</td>
                                                    <td></td>
                                                    <td style="text-align: center">8</td>
                                                    <td>BBa_K2997009 cDNA</td>
                                                    <td style="text-align: center">3+4</td>
                                                </tr>
                                                <tr>
                                                    <td style="text-align: center">2</td>
                                                    <td>BBa_K2997009 cDNA</td>
                                                    <td style="text-align: center">1+2</td>
                                                    <td style="text-align: center">9</td>
                                                    <td>BBa_K2997010 cDNA</td>
                                                    <td style="text-align: center">3+4</td>
                                                </tr>
                                                <tr>
                                                    <td style="text-align: center">3</td>
                                                    <td>BBa_K2997009 RNA</td>
                                                    <td style="text-align: center">1+2</td>
                                                    <td style="text-align: center">10</td>
                                                    <td>BBa_K2997009 RNA</td>
                                                    <td style="text-align: center">3+4</td>
                                                </tr>
                                                <tr>
                                                    <td style="text-align: center">4</td>
                                                    <td>BBa_K2997009 plasmid</td>
                                                    <td style="text-align: center">1+2</td>
                                                    <td style="text-align: center">11</td>
                                                    <td>BBa_K2997010 RNA</td>
                                                    <td style="text-align: center">3+4</td>
                                                </tr>
                                                <tr>
                                                    <td style="text-align: center">5</td>
                                                    <td>BBa_K2997010 cDNA</td>
                                                    <td style="text-align: center">1’+2</td>
                                                    <td style="text-align: center">12</td>
                                                    <td>BBa_K2997009 plasmid</td>
                                                    <td style="text-align: center">3+4</td>
                                                </tr>
                                                <tr>
                                                    <td style="text-align: center">6</td>
                                                    <td>BBa_K2997010 RNA</td>
                                                    <td style="text-align: center">1’+2</td>
                                                    <td style="text-align: center">13</td>
                                                    <td>BBa_K2997010 plasmid</td>
                                                    <td style="text-align: center">3+4</td>
                                                </tr>
                                                <tr>
                                                    <td style="text-align: center">7</td>
                                                    <td>BBa_K2997010 plasmid</td>
                                                    <td style="text-align: center">1’+2</td>
                                                    <td></td>
                                                    <td></td>
                                                    <td></td>
                                                </tr>
                                            </tbody>
                                        </table>
                                    </div>
                                    <figcaption>Fig. 2. Reverse Transcription (RT)-PCR Results to confirm that our construct is being transcribed. (a) Schematics show location of amplified regions and primers. (b) 1.5 % Agarose gel shows PCR results. All products have expected size of 250 bp as shown in (a). Template and primers used in this experiment are listed in the table. (cDNA: Total cDNA; RNA: Total RNA; plasmid: pSB1C3 containing respective Biobrick.)</figcaption>
                                </div>
                            </div>
                        </div>

                        <p><p>    Then, we carried out SDS-PAGE to check the protein expression of TAL, both with TyrP and without TyrP. The expected protein size of TAL is 54 kDa and the expected protein size of TyrP is 43 kDa. As seen in the results below, however, there’s no distinguishable band around both sizes.</p>
                        <div class="container-fluid p-0">
                            <div class="row no-gutters">
                                <div class="col-lg ">
                                    <figure class="d-flex flex-column justify-content-center align-items-center px-lg-3">
                                        <!-- put link in href &amp; src , width in a can adjust-->
                                        <a href="https://static.igem.org/mediawiki/2019/0/02/T--NCKU_Tainan--Result-SDSPAGE_2.jpg" target="_blank" style="width:50%"><img src="https://static.igem.org/mediawiki/2019/0/02/T--NCKU_Tainan--Result-SDSPAGE_2.jpg" alt="" title="" style="width:100%"></a>
                                        <figcaption>Fig. 3. 12% SDS PAGE of <i>E. coli</i> Nissle 1917 with different plasmids. M: Marker; Lane 1: Wild Type; Lane 2: pSB1C3; Lane 3: BBa_K2997009 ; Lane 4: BBa_K2997010; Lane 5: Dual plasmid containing BBa_K2997009 and BBa_K2997000; Lane 6: Dual plasmid containing BBa_K29970010  and BBa_K2997000 ; Lane 7: Positive control (c.d. 3392)</figcaption>
                                    </figure>
                                </div>
                            </div>
                        </div>
                        <p>    Finally, to confirm the protein activity of TAL and TyrP, we performed a functional test using <a href="https://2019.igem.org/Team:NCKU_Tainan/Design#Subtitle1" target="_blank"><i>n</i>-octanol extraction method</a>, which was previously proposed by iGEM_Uppsala 2013 and has been verified by HPLC<sup>[<a href="#ref3">3</a>]</sup>. The <i>p</i>-Coumaric acid concentration was measured through the absorbance value at 310 nm wavelength under Nanodrop UV-Vis wavelength. </p>
                        <div class="container-fluid p-0">
                            <div class="row no-gutters">
                                <div class="col-lg ">
                                    <figure class="d-flex flex-column justify-content-center align-items-center px-lg-3">
                                        <!-- put link in href &amp; src , width in a can adjust-->
                                        <a href="https://static.igem.org/mediawiki/2019/d/d3/T--NCKU_Tainan--Results_tyrosine_to_pCA%2C_RBS_improve.png" target="_blank" style="width:50%"><img src="https://static.igem.org/mediawiki/2019/d/d3/T--NCKU_Tainan--Results_tyrosine_to_pCA%2C_RBS_improve.png" alt="" title="" style="width:100%"></a>
                                        <figcaption>Fig. 4. <i>p</i>-Coumaric acid/OD600 levels of <i>E. coli</i> Nissle with TAL and tyrP in LB with 1mM tyrosine incubated for 48 hours.</figcaption>
                                     </figure>
                                </div>
                            </div>
                        </div>

                        <p>    We compared the TAL constructs containing the native and B0034 ribosome binding sites, (<a href="http://parts.igem.org/Part:BBa_K2997009" target="_blank">BBa_K2997009</a> and <a href="http://parts.igem.org/Part:BBa_K2997010" target="_blank">BBa_K2997010</a>) to determine if <i>p</i>-Coumaric Acid production is improved by changing the ribosome binding sites. From the results seen in Fig. 4, <a href="http://parts.igem.org/Part:BBa_K2997010" target="_blank">BBa_K2997010</a> is able to produce a higher amount of <i>p</i>-Coumaric acid. Hence, we are able to prove that by changing the RBS (from Native to B0034), the conversion of tyrosine into <i>p</i>-Coumaric acid can increase by 1.73-fold. Therefore, we have shown that we have improved a previous BioBrick.</p>
                        <p>    For more information, please visit our <a href="https://2019.igem.org/Team:NCKU_Tainan/Results" target="_blank">Results page.</a></p>
                </section>
                
                <section class="ref">
                    <h2>References</h2>
                    <ol>
            <!--id name must corespond -->
                        <li id="ref1">https://2007.igem.org/wiki/index.php/Edinburgh/Team </li>
                        <li id="ref2">Chen, H., Bjerknes, M., Kumar, R., &amp; Jay, E. (1994). Determination of the optimal aligned spacing between the Shine – Dalgarno sequence and the translation initiation codon of <i>Escherichia coli</i>  mRNAs. <i>Nucleic Acids Research, 22</i>(23), 4953–4957. </li>
                        <li id="ref3">https://2013.igem.org/Team:Uppsala</li>
                    </ol>
                </section>
            </main>
        </div>
    </div>
<script src="https://2019.igem.org/Team:NCKU_Tainan/js/jquery-3_4_1_min_js?action=raw&amp;ctype=text/javascript"></script>
<script src="https://2019.igem.org/Team:NCKU_Tainan/js/4.3.1/bootstrap_min_js?action=raw&amp;ctype=text/javascript"></script>
<script src="https://2019.igem.org/Team:NCKU_Tainan/js/Universal?action=raw&amp;ctype=text/javascript"></script>
<script>
// remember to edit this according to page
$( document ).ready(function() {
$("#breadtext").html('<a href="https://2019.igem.org/Team:NCKU_Tainan/Parts">Parts</a> > <a>Improvement</a>')
});
</script>
</body>

{% endblock %}
