{% extends "layout.html" %}

{% block page_content %}

<style>

@font-face
    {
        font-family:'font_one';
        src:url(https://static.igem.wiki/teams/4242/wiki/acme-regular.ttf)
    }
    .p1
    {
        font-family:'font_one';
        font-size:1.5rem;
        color:#f0f0f0;
    }
    
    @font-face
    {
        font-family:'font_two';
        src:url(https://uploads.igem.org/teams/4242/wiki/Overlock-Regular.ttf)

    }
    .o
    {
        font-family:'font_two';
        font-size:1rem;
        color:#f0f0f0;
    }
    .z1
    {
        font-family:'font_one';
        font-size:1.5rem;
        color:#f26b1f
    }
    .z2
    {
        font-family:'font_two';
        font-size:0.5rem;
        color:#f0f0f0
    }
    
    body
   {
        width:100%;
        height: auto;
        background-color: #21373d;
        overflow-y: scroll;
    }
</style>


<head>
    <meta charset="UTF-8" />
    <title>Team:CUG-China/Notebook - 2022.igem.org</title>
    <meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0,user-scalable=no">
    <link rel="stylesheet" href="{{ url_for('static', filename = 'headama.css') }}">
    <link rel="stylesheet" href="{{ url_for('static', filename = 'notebook amazing/nb.css') }}">
</head>

<body >
    <section class="section main-banner" id="top" data-section="Intro">
        <img src="https://static.igem.wiki/teams/4242/wiki/background/b-team.png" alt="">
        <div class="header-text">
            <div class="caption">
                <div class="wave-text">
                    <span>Model</span>
                </div>
            </div>
        </div>
    </section>

    <div class="container">

          {% include "sider.html" %}
          <aside class="aside">
            <br>
            <div>
              <a class="pointer" href="#introduction">Introduction</a>
              <a class="pointer" href="#modeling">Modeling</a>
              <a class="pointer" href="#model solution">Model Solution</a>
              <a class="pointer" href="#sensitivity analysis">Sensitivity Analysis</a>
              <a class="pointer" href="#references">References</a>
            </div>
          </br>
          </aside>
      
        <div class="row" id="move20">     
          <div class="col" id="Content">
            <!---Introduction--->
             <p class="target-fix" id="introduction"></p>
             <p style="margin-top: 50px;"></p>
             <div class="p1">
            <h1 id="Introduction;" style="color: #243136;"><b>Introduction</b></h1> 
            </div> 
            <div class="o">
            <hr style="border-top:5px solid gray;">
            <ul>
                <p><b>C-di-GMP</b> is the main substance regulating the formation of the microbial membranes, and lowconcentration of <b>c-di-GMP</b> results in the dissociation of biofilm. Better description of the tran-scription and translation processes will play an significant role in he development of a biosenorthat can be widely used for different bacterial <b>c-di-GMP</b> concentrations within bacteria.</p>       
                <p>To solvethis problem, we adopted the repressor-based biosensor model to better describe the transcriptionand translation processes by the set of ordinary differential equations and solve our model by theRunge-Kutta method. All in all, we anticipate that this section of our work could be conducive tothe development of biofilm dissociators, so as to address biofilm triggered and refractory infections.</p>         
            </ul>
            </hr>
            </div>
          <!---Modeling--->
          <p class="target-fix" id="modeling"></p>
          <p style="margin-top: 50px;"></p>
        <div class="p1">
         <h1 id="Modeling;"  style="color: #f26b1f;"><b>Modeling</b></h1> 
         </div> 
         <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
            <p><b>FleQ</b> controls gene expression by a novel mechanism that involves binding to two sites in thepromoter of the operon it controls. At one site (box 2), <b>FleQ</b> functions to repress gene expression,and at the other site (box 1), <b>FleQ</b> functions to activate gene expression in response to <b>c-di-GMP</b>.A revised model for pel regulation by <b>FleQ</b> can be proposed based on the data presented here.In the absence of <b>c-di-GMP</b>, <b>FleQ</b> binds to its two <b>FleQ</b> binding sites on the pel promoter.</p>       
            <p><b>FleN</b> is found bound to <b>FleQ</b>. In the presence of ATP, we propose that FleN forms dimers inducing abending of the pelA promoter by bridging the bound <b>FleQ proteins</b> . We think that this conformationcould either impair the binding of the RNA polymerase or prevent RNA polymerase bound to the pelA promoter from forming an open complex, leading to pel repression.</p>
            <p>In the presence of c-di-GMP, FleQ undergoes a conformational change that probably induces a cascade of conformationalchanges in the FleQ/FleN/DNA complex such that the bending is relaxed. FleQ is switched to anactivator. The relief of the bending may either induce RNA polymerase binding or remodel RNApolymerase binding and the FleQ bound to FleQ box 1 favors transcription initiation.</p>
            <p>This modelis speculative and still incomplete, but it is consistent with the ability of FleQ to mediate bothrepression and activation in response to c-di-GMP while bound at the same sites on DNA[4].	In our whole model, we have taken the assumptions as follows.</p>
            <b><p>In our whole model, we have taken the assumptions as follows.</p></b>
            <p>1.The consumption of ATP and translation resources takes the Michaelis-Menten kinetics, so the kinetics should satisfy Eq.2.7-2.8.</p>
            <p>2.The value of the concentration of c-di-GMP keeps constant.</p>
            <p>3.No inhibitory effects of any species.</p>
            <p>4.The concentration of all genes remain constant.</p>
            <p>5.Michaelis-Menten kinetics occurs under quasi steady-state.</p>
            <p>6.No diffusion limitations and evaporation.</p>
            <p>Firstly, we gave the set of differential equations. transcription of FleQ</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model1.png" alt="Pulpit rock" width="960" height="120"> 
            </p>
            <p>translation of FleQ</p>  
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model2.png" alt="Pulpit rock" width="960" height="120"> 
            </p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model3.png" alt="Pulpit rock" width="551" height="425"> 
            </p>
            <p>Figure 1: Model of pel regulation. This figure is from reference [4].</p>
            <p>transcription of FleN</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model4.png" alt="Pulpit rock" width="1000" height="120"> 
            </p>
            <p>translation of FleN</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model5.png" alt="Pulpit rock" width="1100" height="120"> 
            </p>
            <p>transcription of gfp</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model6.png" alt="Pulpit rock" width="960" height="120"> 
            </p>
            <p>translation of gfp</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model7.png" alt="Pulpit rock" width="1200" height="120"> 
            </p>
            <p>consumption of TLR</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model8.png" alt="Pulpit rock" width="890" height="120"> 
            </p>
            <p>consumption of ATP</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model9.png" alt="Pulpit rock" width="900" height="120"> 
            </p>
            <p>consumption of c-di-GMP</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model10.png" alt="Pulpit rock" width="800" height="120"> 
            </p>
            <p>FleN forms dimers</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model11.png" alt="Pulpit rock" width="1200" height="120"> 
            </p>
            <p>We adopt the repressor-based biosensor model in [3] and extend it to the FleQ-FleN-DNA combined system to analyze the expression and these genes by calculating the concentration of fluorescence protein. The set of ordinary differential equations in Eq.2.1-2.10 have described our model.</p>
            <p>Eq.2.1,2.3,2.5 describe the transcription process of FleQ, FleN and gfp respectively, where
                νf leQ, νf leN and νg f p stand for the transcription rate constants of FleQ, FleN and gfp whileλf leQ2, λf leN2 and λg f p are FleQ,fleN and gfp mRNA degradation rate constants. Furthermore, KT X2 is the
                Michaelis-Menten constant for transcription. Gf leQ, Gf leN and Gg f p are genes.</p>
            <p>Eq.2.2,2.4,2.6 describe the translation process of FleQ, FleN and gfp, respectively where kTL2is the translation rate constant for all proteins and KTL2 the Michaelis-Menten constant for thetranslation process of gfp, FleQ and FleN. Moreover, For more simplicity, we set the consumptionprocess of translation resources as a Michaelis-Menten process. In the translation process of FleN,we took the concentration of FleN monomer and dimer as repression and promoter factors forthe translation process of FleN. k2 fleN and k−2 fleN are Repressor FleN dimerization and dimerdissociation rate constants. The effects of FleQ monomer and FleN dimer on the translation processof gfp are also consider. kfleQ−gf p is the effect constant of FleQ on the translation of gfp andkdimer−g f p the effect constant of dimer fleN2 on the translation of gfp.</p>
            <p>Eq.2.7,2.8 describe the consumption rate of translation resources and ATP, they both followthe Michaelis-Menten equation, but in Eq.2.9, we set the concentration of c-di-GMP to be constantto investigate the influence of the concentration of c-di-GMP on the gene expression process better.νλTLR and νλATP are consumption rate constants of translation resources and ATP. KλTLR and KλATPare Michaelis-Menten constants for the consumption of translation resources and ATP.
            </p>
        </ul>
      </hr>
        </div>
<!---model solution--->
          <p class="target-fix" id="model solution"></p>
          <p style="margin-top: 50px;"></p>
        <div class="p1">
         <h1 id="model solution;"  style="color: #f26b1f;"><b>model solution</b></h1> 
         </div> 
         <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
           <p>We solved our model using the Runge-Kutta method, which is one of the most frequently usedhigh-precision methods to solve ordinary differential equations, and this kind of method has arelated relationship with Taylor series.
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/12.png" alt="Pulpit rock" width="572" height="94"> 
           </p>
           <p>and as long as the function f(x, y) satisfied the Lipschitz condition</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/13.png" alt="Pulpit rock" width="667" height="70"> 
           </p>
           <p>Eq. will have a theoretical solution y = y(x) and it is the only solution.</p>
           <p>
            Table 1: Overview of all species used in our deterministic model. All species’ concentration units
are all [nM], and we gave a brief description.
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/14.png" alt="Pulpit rock" width="804" height="417"> 
           </p>
           <p>Table 2: Overview of all parameter values in the deterministic kinetic model. The parameter constants, their values and units, and a description is shown.
            </p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/15.png" alt="Pulpit rock" width="1110" height="190"> 
            </p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/16.png" alt="Pulpit rock" width="880" height="760"> 
            </p>
           <p>Eq. will have a theoretical solution y = y(x) and it is the only solution.</p>
           <p>Assume the solution y = y(x), and according to the Taylor expansion
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/17.png" alt="Pulpit rock" width="777" height="84">
           </p>
           <p>where h is the step length, and then we can draw the Taylor equation
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/18.png" alt="Pulpit rock" width="710" height="110">
           </p>
           <p>It is obvious that the first-order Taylor equation is
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/19.png" alt="Pulpit rock" width="608" height="79">
           </p>
           <p>which is used commonly in some rough estimations.
           </p>
           <p>The essence of Runge-Kutta method is to employ the Taylor series technique and investigate the difference quotient y(xn+1)−y(xn)an appropriate 0 < θ < 1 to make	, and according to the differential mean value theorem, there is
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/20.png" alt="Pulpit rock" width="668" height="90">
           </p>
          <p>The simplest one is the slope given by Euler’s formula, which approximately takes the slope atpoint xn as K1 = f(xn,yn), but this slope has low level of precision. The improved Euler’s formula is
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/201.png" alt="Pulpit rock" width="692" height="165">
           </p>
          <p>
            where we have taken the mean value of the slopes at two points [xn,xn+1], so we have a more preciseresult. In the same way, after calculating more points and get their slopes, we can get a formulawith higher precision. Taking this thought, we can get the fourth-order Runge-Kutta formula
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/21.png" alt="Pulpit rock" width="750" height="270">
           </p>
          <p>
            The fourth-order Runge-Kutta method needs to calculate the function f for four times, and it hasan error of O(h5). If we take the step length h to be small enough, we could get a result with veryhigh level of precision.
          </p>The Eq.3.8 is the formula we take in our model, and it can be easily used in our ordinarydifferential equations Eq.2.1-2.10.
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/22.png" alt="Pulpit rock" width="700" height="538">
           </p>
          <p>
            Figure 3: A plot of the concentration of fluorescence with respect to time. Blue, red and yellowlines stand for the different concentrations of initial c-di-GMP which are 3 nM, 30 nM and 300 nM,respectively.
          </p>
          <p>
            After solving this set of equations, we got the gfp fluorescence over time shown in Fig.3. Itis clear from this figure that for low-concentration c-di-GMP, the concentration of fluorescenceprotein decreases steadily, and for high-concentration c-di-GMP, the concentration of fluorescenceprotein decreases first and then increases. If the concentration of c-di-GMP is high enough, theultimate concentration of fluorescence may be higher than the initial value.
          </p>
          <p>
            And how about the influence of the concentration of FleN on the concentration of fluores-cence? According to our experiments and mechanism analysis, the FleN is one kind of repressorprotein, so the concentration of fluorescence will be decreased. Fig.4 shows the final concentrationof fluorescence with respect to the concentration of FleN. It is clear from this figure that the FleNhas a repression effect on the expression of fluorescence protein gfp. It should be noticed that inour experimental setup, FleN is not necessary to repress the expression of gfp, when the concentra-tion of c-di-GMP is appropriate, the concentration of fluorescence could also be increased withoutFleN.
          </p>
          <p>
            Similarly, the concentration of fluorescence will decrease with higher concentration of FleQ.Fig.5 shows the change of the concentration of gfp over time for different concentrations of FleN.
          </p>
        </ul>
      </hr>
        </div>
<!---sensitivity analysis--->
          <p class="target-fix" id="sensitivity analysis"></p>
          <p style="margin-top: 50px;"></p>

        <div class="p1">
         <h1 id="sensitivity analysis;"  style="color: #f26b1f;"><b>Sensitivity Analysis</b></h1> 
         </div> 
        <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
          <p>
            Our goal here is to get the concentration curve of fluorescence over time to understand the under-lying mechanism of the biofilm dissociation process, but under actual experimental conditions, the parameters in our model may be some different, so it is necessary to investigate the sensitivity ofour model with respect to all parameters.
          </p>
          <p>f
            There are a total of 22 parameters in our model, and it would be time-consuming and tediousto analyze these all parameters one by one. We took the celebrated Morris method which is a globalsensitivity analysis method in statistical subjects. This method could be divided into three steps:	1. Set up definite ranges for all input parameters, and give all parameters definite valuesrandomly in these ranges. Calculate the result of model.
          </p>
          <p>
            2. Change the value for one input parameter and calculate the resulting change in modeloutcome compared to its first run.
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/23.png" alt="Pulpit rock" width="288" height="265">
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/24.png" alt="Pulpit rock" width="381" height="270">
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/25.png" alt="Pulpit rock" width="765" height="535">
           </p>
          <p>
            3. Change all input parameters one by one and calculate the result changes compared to thelast run after every change.
          </p>
          <p>
            4. Repeat step 1-3 for r times.
          </p>
          <p>
            Assume there are k parameters in total, then there should be r(k+1) runs in total. if Ei is theresult change for changing parameter i, and there will be r Ei j in total. The mean value and standarddeviation for every parameter can be calculated
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/26.png" alt="Pulpit rock" width="643" height="186">
           </p>
          <p>
            Actually, the mean value for the absolute value of every result change can describe the sensi-
tivity better, so we define
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/27.png" alt="Pulpit rock" width="619" height="94">
           </p>
         <p>
          In our analysis, the starting values for all parameters should between 0.8 times and 1.2 timesof their original values in table 2, and their ranges are shown in table.3. In every step, we increasedor decreased the value of one parameter by 1%.
         </p>
         <p>
          After calculation, we chose the absolute value of µi when time = 10 min, and our parameterscan be divided into four categories, which is shown in table.4.
         </p>
         <p>
          The mean values µ of all parameters are shown in Fig.6. For parameters in sensitivity level 1,a 1% change of the value of one parameter only leads to a decrease or increase in the value of theconcentration of gfp by no more than 0.001% for most time points, and for parameters in sensitivitylevel 4, a 1% change of the value of one parameter leads to a change of no more than 0.5%.	Fig.7 shows the values of µ⋆ for all parameters. The quantity µ⋆ is actually more meaningfulcause it reflects the real change for all parameters. The values of µ⋆ are larger than correspondingµ because we calculated the absolute values in every step of change. It is clear that the outcomemay be sensitive to some parameters like kTL2, KTL2 and kdimer−g f p because a 1% change of thevalue of these parameters lead to a change of the outcome of over 1%.
         </p>
         <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/28.png" alt="Pulpit rock" width="513" height="677">
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/29.png" alt="Pulpit rock" width="435" height="210">
           </p>
         <p>
          How to quantify the sensitivity of our model? We choose the value of the concentration of gfpwhen time = 10 min, which is denoted by cf because the concentration has been stabilized on thispoint. Varying the value of every parameter by 1%, and calculate the change of cf to show the levelof sensitivity.
         </p>
         <p>
          It is clear from table.5 that our model are not sensitive to most parameters since an increase ordecrease of 1% of these parameters just lead to a quite small change of cf . Among all parameters,kTL2, KTL2 and kGMP−dimer are relatively sensitive. We plotted the curves after changing kTL2 and kGMP−dimer compare it with the original one in Fig.9.
         </p>
         <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/30.png" alt="Pulpit rock" width="427" height="287">
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/31.png" alt="Pulpit rock" width="427" height="293">
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/32.png" alt="Pulpit rock" width="334" height="268">
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/33.png" alt="Pulpit rock" width="394" height="283">
           </p>

         </ul>
        </hr>
        </div>
        
        
      </hr>
      </div> 
      <!---References--->
          <p class="target-fix" id="references"></p>
          <p style="margin-top: 50px;"></p>
        <div class="p1">
         <h1 id="references;"  style="color: #f26b1f;"><b>References</b></h1> 
         </div> 
         <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
          <p class="z2">[1] Stögbauer T, Windhager L, Zimmer R, et al. Experiment and mathematical modeling of gene	expression dynamics in a cell-free system[J]. Integrative Biology, 2012, 4(5): 494-501.</p>
          <p class="z2">[2] Stamatakis M, Mantzaris N V. Comparison of deterministic and stochastic models of the lac operon	genetic network[J]. Biophysical journal, 2009, 96(3): 887-906.</p>
          <p class="z2">[3] Gräwe A, Dreyer A, Vornholt T, et al. A paper-based, cell-free biosensor system for the detection of	heavy metals and date rape drugs[J]. PloS one, 2019, 14(3): e0210940.</p>
          <p class="z2">[4] Baraquet C, Murakami K, Parsek M R, et al. The FleQ protein from Pseudomonas aeruginosafunctions as both a repressor and an activator to control gene expression from the pel operonpromoter in response to c-di-GMP[J]. Nucleic acids research, 2012, 40(15): 7207-7218.</p>
          <p class="z2">[5] Baraquet C, Harwood C S. Cyclic diguanosine monophosphate represses bacterial flagella synthesisby interacting with the Walker A motif of the enhancer-binding protein FleQ[J]. Proceedings of theNational Academy of Sciences, 2013, 110(46): 18478-18483.</p>
          <p class="z2">[6] Gentner M, Allan M G, Zaehringer F, et al. Oligomer formation of the bacterial second messengerc-di-GMP: reaction rates and equilibrium constants indicate a monomeric state at physiologicalconcentrations[J]. Journal of the American Chemical Society, 2012, 134(2): 1019-1029.</p>
          <p class="z2">[7] Stamatakis M, Mantzaris N V. Comparison of deterministic and stochastic models of the lac operon	genetic network[J]. Biophysical journal, 2009, 96(3): 887-906.</p>
          <p class="z2">[8] Saltelli A, Ratto M, Andres T, et al. Global sensitivity analysis: the primer[M]. John Wiley and Sons,	2008.</p>
          <p class="z2">[9] Karzbrun E, Shin J, Bar-Ziv R H, et al. Coarse-grained dynamics of protein synthesis in a cell-free	system[J]. Physical review letters, 2011, 106(4): 048104.</p>           
        </ul>
      </hr>
        </div>
        </div>
        </div>
<body>

{% endblock %}
