{% extends "layout.html" %}

{% block page_content %}

<style>

@font-face
    {
        font-family:'font_one';
        src:url(https://static.igem.wiki/teams/4242/wiki/acme-regular.ttf)
    }
    .p1
    {
        font-family:'font_one';
        font-size:1.5rem;
        color:#f0f0f0;
    }
    
    @font-face
    {
        font-family:'font_two';
        src:url(https://uploads.igem.org/teams/4242/wiki/Overlock-Regular.ttf)

    }
    .o
    {
        font-family:'font_two';
        font-size:1.2rem;
        color:#f0f0f0;
    }
    .z1
    {
        font-family:'font_one';
        font-size:1.5rem;
        color:#f26b1f
    }
    .z2
    {
        font-family:'font_two';
        font-size:0.7rem;
        color:#f0f0f0
    }
    .z3
    {
        font-family:'font_one';
        font-size:1.5rem;
        color:#f0f0f0;
    }
    body
   {
        width:100%;
        height: auto;
        background-color: #21373d;
        overflow-y: scroll;
    }
    #move20
    {
      margin-left: 20%;
    }

    @media screen and (max-width: 992px) 
    {
      #move20 {
      margin-left: 0%;
    }
</style>


<head>
    <meta charset="UTF-8" />
    <title>Team:CUG-China/Notebook - 2022.igem.org</title>
    <meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0,user-scalable=no">
    <link rel="stylesheet" href="{{ url_for('static', filename = 'headama.css') }}">
    <link rel="stylesheet" href="{{ url_for('static', filename = 'notebook amazing/nb.css') }}">
</head>

<body >
    <section class="section main-banner" id="top" data-section="Intro">
        <img src="https://static.igem.wiki/teams/4242/wiki/background/b-part.png" alt="">
        <div class="header-text">
            <div class="caption">
                <div class="wave-text">
                    <span>Model</span>
                </div>
            </div>
        </div>
    </section>

    <div class="container">

          {% include "sider.html" %}
          <aside class="aside">
            <br>
            <div>
              <a class="pointer" href="#introduction">Introduction</a>
              <a class="pointer" href="#Part I">Part I</a>
              <a class="pointer" href="#Part II">Part II</a>
              <a class="pointer" href="#Part III">Part III</a>
              <a class="pointer" href="#references">References</a>
            </div>
          </br>
          </aside>
      
        <div class="row" id="move20">     
          <div class="col" id="Content">
            <!---Introduction--->
             <p class="target-fix" id="introduction"></p>
             <p style="margin-top: 50px;"></p>
             <div class="p1">
            <h1 id="Introduction;" style="color: #f26b1f"> Introduction </h1> 
            </div> 
            <div class="o">
            <hr style="border-top:5px solid gray;">
            <ul>
                <p> C-di-GMP  is the main substance regulating the formation of the biofilm, and low concentration of  c-di-GMP  results in the dissociation of biofilm. Better description of the tran-scription and translation processes will play an significant role in he development of a biosensor that can be widely used for different bacterial  c-di-GMP  concentrations within bacteria.</p>       
                <p>To solve this problem, we adopted the transcription-based biosensor model to better describe the transcription and translation processes by the set of ordinary differential equations and solve our model by the Runge-Kutta method. All in all, we anticipate that this section of our work could be conducive to the development of biofilm dissociators, so as to address biofilm triggered and refractory infections.</p>         
            </ul>
            </hr>
            </div>
          <!---Modeling--->
          <p class="target-fix" id="modeling"></p>
          <p style="margin-top: 50px;"></p>
        <div class="p1">
         <h1 id="Modeling;"  style="color: #f26b1f">Part I Modeling </h1> 
         </div> 
         <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
            <p> FleQ  controls gene expression by a novel mechanism that involves binding to two sites in thepromoter of the operon it controls. At one site (box 2),  FleQ  functions to repress gene expression,and at the other site (box 1),  FleQ  functions to activate gene expression in response to  c-di-GMP .A revised model for <i>pel</i> regulation by  FleQ  can be proposed based on the data presented here. In the absence of  c-di-GMP ,  FleQ  binds to its two  FleQ  binding sites on the <i>pel</i> promoter.</p>       
            <p> FleN  is found bound to  FleQ . In the presence of ATP, we propose that FleN forms dimers inducing abending of the <i>pelA</i> promoter by bridging the bound  FleQ proteins. We think that this conformationcould either impair the binding of the RNA polymerase or prevent RNA polymerase bound to the <i>pelA</i> promoter from forming an open complex, leading to <i>pel</i> repression.</p>
            <p>In the presence of c-di-GMP, FleQ undergoes a conformational change that probably induces a cascade of conformational changes in the FleQ/FleN/DNA complex such that the bending is relaxed. FleQ is switched to anactivator. The relief of the bending may either induce RNA polymerase binding or remodel RNA polymerase binding and the FleQ bound to FleQ box 1 favors transcription initiation.</p>
            <p>This modelis speculative and still incomplete, but it is consistent with the ability of FleQ to mediate both repression and activation in response to c-di-GMP while bound at the same sites on DNA<sup>[4]</sup>.	In our whole model, we have taken the assumptions as follows.</p>
            <p>1.The consumption of ATP and translation resources takes the Michaelis-Menten kinetics, so the kinetics should satisfy Eq.2.7-2.8.</p>
            <p>2.The value of the concentration of c-di-GMP keeps constant.</p>
            <p>3.No inhibitory effects of any species.</p>
            <p>4.The concentration of all genes remain constant.</p>
            <p>5.Michaelis-Menten kinetics occurs under quasi steady-state.</p>
            <p>6.No diffusion limitations and evaporation.</p>
            <p>Firstly, we gave the set of differential equations.</p>
            <p>transcription of <i>FleQ</i></p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model1.png" alt="Pulpit rock" width="480" height="60"> 
            </p>
            <p>translation of <i>FleQ</i></p>  
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model2.png" alt="Pulpit rock" width="480" height="60"> 
            </p>
            <p>transcription of <i>FleN</i></p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model4.png" alt="Pulpit rock" width="500" height="60"> 
            </p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model3.png" alt="Pulpit rock" width="551" height="425"> 
            </p>
            <p style="text-align: center" class="z2">Figure 1: Model of pel regulation. This figure is from reference <sup>[4]</sup>.</p>
            <p>translation of <i>FleN</i></p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model5.png" alt="Pulpit rock" width="550" height="60"> 
            </p>
            <p>transcription of <i>gfp</i></p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model6.png" alt="Pulpit rock" width="480" height="60"> 
            </p>
            <p>translation of <i>gfp</i></p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model7.png" alt="Pulpit rock" width="600" height="60"> 
            </p>
            <p>consumption of TLR</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model8.png" alt="Pulpit rock" width="445" height="60"> 
            </p>
            <p>consumption of ATP</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model9.png" alt="Pulpit rock" width="450" height="60"> 
            </p>
            <p>consumption of c-di-GMP</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model10.png" alt="Pulpit rock" width="400" height="60"> 
            </p>
            <p>FleN forms dimers</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/model11.png" alt="Pulpit rock" width="600" height="60"> 
            </p>
            <p>We adopt the repressor-based biosensor model in <sup>[3]</sup> and extend it to the FleQ-FleN-DNA combined system to analyze the expression and these genes by calculating the concentration of fluorescence protein. The set of ordinary differential equations in Eq.2.1-2.10 have described our model.</p>
            <p>Eq.2.1,2.3,2.5 describe the transcription process of FleQ, FleN and gfp respectively, where
                ν<span class="z2">fleQ</span>, ν<span class="z2">fleN</span> and ν<span class="z2">gfp</span> stand for the transcription rate constants of FleQ, FleN and gfp while λ<span class="z2">fleQ2</span>, λ<span class="z2">fleN2</span> and λ<span class="z2">gfp</span> are FleQ,fleN and gfp mRNA degradation rate constants. Furthermore, K<span class="z2">TX</span>2 is the
                Michaelis-Menten constant for transcription. G<span class="z2">fleQ</span>, G<span class="z2">fleN</span> and G<span class="z2">gfp</span> are genes.</p>
            <p>Eq.2.2,2.4,2.6 describe the translation process of FleQ, FleN and gfp, respectively where k<span class="z2">TL2</span> is the translation rate constant for all proteins and K<span class="z2">TL2</span> the Michaelis-Menten constant for the translation process of gfp, FleQ and FleN. Moreover, For more simplicity, we set the consumption process of translation resources as a Michaelis-Menten process. In the translation process of FleN, we took the concentration of FleN monomer and dimer as repression and promoter factors for the translation process of FleN. k<span class="z2">2fleN</span> and k<span class="z2">_2fleN</span> are Repressor FleN dimerization and dimerdissociation rate constants. The effects of FleQ monomer and FleN dimer on the translation processof gfp are also consider. k<span class="z2">fleQ_gfp</span> is the effect constant of FleQ on the translation of gfp and k<span class="z2">dimer_gfp</span> the effect constant of dimer fleN<span class="z2">2</span> on the translation of <i>gfp</i>.</p>
            <p>Eq.2.7,2.8 describe the consumption rate of translation resources and ATP, they both follow the Michaelis-Menten equation, but in Eq.2.9, we set the concentration of c-di-GMP to be constant to investigate the influence of the concentration of c-di-GMP on the gene expression process better. ν<span class="z2">λTLR</span> and ν<span class="z2">λATP</span> are consumption rate constants of translation resources and ATP. K<span class="z2">λTLR</span> and K<span class="z2">λATP</span> are Michaelis-Menten constants for the consumption of translation resources and ATP.
            </p>
            <p>Finally, in Eq. 2.10, we model the dimerization process of fleN with the consideration of the effects of ATP (promoter) and c-di-GMP (repressor). The prefactor [GMP] accounts for the saturation effect of c-di-GMP. k<span class="z2">ATP_fleN</span> and k<span class="z2">GMP_dimer</span> stand for the effect constant of ATP on the dimeriazation process of fleN and effect constant of c-di-GMP on the dimeriazation process of fleN, respectively.</p>
            <p>The overall reaction can be illustrated by Fig.2.</p>
        </ul>
      </hr>
        </div>
<!---model solution--->
          <p class="target-fix" id="model solution"></p>
          <p style="margin-top: 50px;"></p>
        <div class="p1">
         <h1 id="model solution;"  style="color: #f26b1f;">Part II model solution </h1> 
         </div> 
         <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
           <p>We solved our model using the Runge-Kutta method, which is one of the most frequently usedhigh-precision methods to solve ordinary differential equations, and this kind of method has arelated relationship with Taylor series.</p>
           <p>To introduce the Runge-Kutta method, we first induce the Taylor series method: consider the first-order ordinary differential equation's initial value</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/12.png" alt="Pulpit rock" width="365" height="60"> 
           </p>
           <p>and as long as the function f(x, y) satisfied the Lipschitz condition</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/13.png" alt="Pulpit rock" width="572" height="60"> 
           </p>
           <p>Eq. will have a theoretical solution y = y(x) and it is the only solution.</p>
           <p class="z2">
            Table 1: Overview of all species used in our deterministic model. All species'concentration units are all [nM], and we gave a brief description.
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/14.png" alt="Pulpit rock" width="578" height="300"> 
           </p>
           <p class="z2">Table 2: Overview of all parameter values in the deterministic kinetic model. The parameter constants, their values and units, and a description is shown.</p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/15.png" alt="Pulpit rock" width="843" height="600"> 
            </p>
            <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/model/16.png" alt="Pulpit rock" width="695" height="600"> 
            </p>
            <p style="text-align: center" class="z2">Fig 2: Illustration of the whole gene expressions process. Rectangles represent species like mRNA, circles represent reactions, dashes lines represent repressions on a reaction, arrows indicate that a species is an educt in a reaction.</p>
             <p>Assume the solution y = y(x), and according to the Taylor expansion
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/17.png" alt="Pulpit rock" width="555" height="60">
           </p>
           <p>where h is the step length, and then we can draw the Taylor equation
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/18.png" alt="Pulpit rock" width="452" height="70">
           </p>
           <p>It is obvious that the first-order Taylor equation is
           </p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/19.png" alt="Pulpit rock" width="462" height="60">
           </p>
           <p>which is used commonly in some rough estimations.
           </p>
           <p>The essence of Runge-Kutta method is to employ the Taylor series technique and investigate the difference quotient y(xn+1)−y(xn)an appropriate 0 < θ < 1 to make	, and according to the differential mean value theorem, there is
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/20.png" alt="Pulpit rock" width="445" height="60">
           </p>
          <p>The simplest one is the slope given by Euler’s formula, which approximately takes the slope atpoint xn as K1 = f(xn,yn), but this slope has low level of precision. The improved Euler’s formula is
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/201.png" alt="Pulpit rock" width="545" height="130">
           </p>
          <p>
            where we have taken the mean value of the slopes at two points [xn,xn+1], so we have a more preciseresult. In the same way, after calculating more points and get their slopes, we can get a formulawith higher precision. Taking this thought, we can get the fourth-order Runge-Kutta formula
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/21.png" alt="Pulpit rock" width="611" height="220">
           </p>
          <p>
            The fourth-order Runge-Kutta method needs to calculate the function f for four times, and it hasan error of O(h5). If we take the step length h to be small enough, we could get a result with veryhigh level of precision.
          </p>The Eq.3.8 is the formula we take in our model, and it can be easily used in our ordinarydifferential equations Eq.2.1-2.10.
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/22.png" alt="Pulpit rock" width="586" height="450">
           </p>
          <p style="text-align: center" class="z2">
            Fig 3: A plot of the concentration of fluorescence with respect to time. Blue, red and yellowlines stand for the different concentrations of initial c-di-GMP which are 3 nM, 30 nM and 300 nM,respectively.
          </p>
          <p>
            After solving this set of equations, we got the <i>gfp</i> fluorescence over time shown in Fig.3. It is clear from this figure that for low-concentration c-di-GMP, the concentration of fluorescence protein decreases steadily, and for high-concentration c-di-GMP, the concentration of fluorescence protein decreases first and then increases. If the concentration of c-di-GMP is high enough, the ultimate concentration of fluorescence may be higher than the initial value.
          </p>
          <p>
            And how about the influence of the concentration of FleN on the concentration of fluores-cence? According to our experiments and mechanism analysis, the FleN is one kind of repressorprotein, so the concentration of fluorescence will be decreased. Fig.4 shows the final concentrationof fluorescence with respect to the concentration of FleN. It is clear from this figure that the FleN has a repression effect on the expression of fluorescence protein <i>gfp</i>. It should be noticed that in our experimental setup, FleN is not necessary to repress the expression of <i>gfp</i>, when the concentra-tion of c-di-GMP is appropriate, the concentration of fluorescence could also be increased without FleN.
          </p>
          <p>
            Similarly, the concentration of fluorescence will decrease with higher concentration of FleQ. Fig.5 shows the change of the concentration of gfp over time for different concentrations of FleN.
          </p>
        </ul>
      </hr>
        </div>
<!---sensitivity analysis--->
          <p class="target-fix" id="sensitivity analysis"></p>
          <p style="margin-top: 50px;"></p>

        <div class="p1">
         <h1 id="sensitivity analysis;"  style="color: #f26b1f;">Part III Sensitivity Analysis </h1> 
         </div> 
        <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
          <p>
            Our goal here is to get the concentration curve of fluorescence over time to understand the under-lying mechanism of the biofilm dissociation process, but under actual experimental conditions, the parameters in our model may be some different, so it is necessary to investigate the sensitivity of our model with respect to all parameters.
          </p>
          <p>
            There are a total of 22 parameters in our model, and it would be time-consuming and tedious to analyze these all parameters one by one. We took the celebrated Morris method which is a globalsensitivity analysis method in statistical subjects. This method could be divided into three steps:	1. Set up definite ranges for all input parameters, and give all parameters definite valuesrandomly in these ranges. Calculate the result of model.
          </p>
          <p>
            2. Change the value for one input parameter and calculate the resulting change in model outcome compared to its first run.
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/23.png" alt="Pulpit rock" width="288" height="265">
           </p>
           <p style="text-align: center" class="z2">Figure 4: The abscissa is the concentration of fleN, and the ordinate is the ultimate concentration of fluorescence when time → ∞.</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/24.png" alt="Pulpit rock" width="381" height="270">
           </p>
           <p style="text-align: center" class="z2">Figure 5: The change of the concentration of gfp over time for different concentrations of fleN.Blue, red and yellow lines stand for the different concentrations of initial c-di-GMP which are 1nM, 5nM and 10nM, respectively.</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/25.png" alt="Pulpit rock" width="765" height="535">
           </p>
           <p style="text-align: center" class="z2">Figure 6: The mean value µ for all parameters in four categories.</p>
          <p>
            3. Change all input parameters one by one and calculate the result changes compared to the last run after every change.
          </p>
          <p>
            4. Repeat step 1-3 for r times.
          </p>
          <p>
            Assume there are k parameters in total, then there should be r(k+1) runs in total. if Ei is the result change for changing parameter i, and there will be r Ei j in total. The mean value and standard deviation for every parameter can be calculated
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/26.png" alt="Pulpit rock" width="346" height="100">
           </p>
          <p>
            Actually, the mean value for the absolute value of every result change can describe the sensitivity better, so we define
          </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/27.png" alt="Pulpit rock" width="395" height="60">
           </p>
         <p>
          In our analysis, the starting values for all parameters should between 0.8 times and 1.2 times of their original values in table 2, and their ranges are shown in table.3. In every step, we increased or decreased the value of one parameter by 1%.
         </p>
         <p>
          After calculation, we chose the absolute value of µi when time = 10 min, and our parameters can be divided into four categories, which is shown in table.4.
         </p>
         <p>
          The mean values µ of all parameters are shown in Fig.6. For parameters in sensitivity level 1,a 1% change of the value of one parameter only leads to a decrease or increase in the value of the concentration of gfp by no more than 0.001% for most time points, and for parameters in sensitivity level 4, a 1% change of the value of one parameter leads to a change of no more than 0.5%.	Fig.7 shows the values of µ⋆ for all parameters. The quantity µ⋆ is actually more meaningful cause it reflects the real change for all parameters. The values of µ⋆ are larger than corresponding µ because we calculated the absolute values in every step of change. It is clear that the outcome may be sensitive to some parameters like kTL2, KTL2 and kdimer−g f p because a 1% change of the value of these parameters lead to a change of the outcome of over 1%.
         </p>
         <p style="text-align: center" class="z2">Table 3: The ranges for all parameters.</p>
         <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/28.png" alt="Pulpit rock" width="513" height="677">
           </p>
           <p style="text-align: center" class="z2">Table 4: Four levels of sensitivity</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/29.png" alt="Pulpit rock" width="435" height="210">
           </p>
           <p>Fig.7 shows the values of µ⋆ for all parameters. The quantity µ⋆ is actually more meaningful cause it reflects the real change for all parameters. The values of µ⋆ are larger than corresponding µ because we calculated the absolute values in every step of change. It is clear that the outcome may be sensitive to some parameters like kT L2, KT L2 and kdimer−g f p because a 1% change of the value of these parameters lead to a change of the outcome of over 1%.</p>
        
         <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/30.png" alt="Pulpit rock" width="427" height="287">
           </p>
           <p style="text-align: center" class="z2">Fig 7: µ⋆ for all parameters in four categories.</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/31.png" alt="Pulpit rock" width="427" height="293">
           </p>
           <p style="text-align: center" class="z2">Fig 8: The standard deviation σ for all parameters in four categories.</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/32.png" alt="Pulpit rock" width="334" height="268">
           </p>
           <p>Fig.8 shows the values of the standard deviations of all parameters. The shapes of all curves are very similar to the curves in Fig.7. σ values for sensitive parameters are larger than those for insensitive parameters.</p>
           <p>How to quantify the sensitivity of our model? We choose the value of the concentration of <i>gfp</i> when time = 10 min, which is denoted by cf because the concentration has been stabilized on this point. Varying the value of every parameter by 1%, and calculate the change of cf to show the level of sensitivity.</p>
           <p>It is clear from table.5 that our model are not sensitive to most parameters since an increase ordecrease of 1% of these parameters just lead to a quite small change of cf . Among all parameters,kTL2, KTL2 and kGMP−dimer are relatively sensitive. We plotted the curves after changing kTL2 and kGMP−dimer compare it with the original one in Fig.9.</p>
           <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/model/33.png" alt="Pulpit rock" width="394" height="283">
           </p>
           <p style="text-align: center" class="z2">Figure 9: The curves after changing kT L2 and kGMP_dimer compared with the original for all parameters.</p>

         </ul>
        </hr>
        </div>
        
        
      </hr>
      </div> 
      <!---References--->
          <p class="target-fix" id="references"></p>
          <p style="margin-top: 50px;"></p>
        <div class="p1">
         <h1 id="references;"  style="color: #f26b1f;"> References </h1> 
         </div> 
         <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
          <p class="z2">[1] Stögbauer T, Windhager L, Zimmer R, et al. Experiment and mathematical modeling of gene	expression dynamics in a cell-free system[J]. Integrative Biology, 2012, 4(5): 494-501.</p>
          <p class="z2">[2] Stamatakis M, Mantzaris N V. Comparison of deterministic and stochastic models of the lac operon	genetic network[J]. Biophysical journal, 2009, 96(3): 887-906.</p>
          <p class="z2">[3] Gräwe A, Dreyer A, Vornholt T, et al. A paper-based, cell-free biosensor system for the detection of	heavy metals and date rape drugs[J]. PloS one, 2019, 14(3): e0210940.</p>
          <p class="z2">[4] Baraquet C, Murakami K, Parsek M R, et al. The FleQ protein from Pseudomonas aeruginosafunctions as both a repressor and an activator to control gene expression from the pel operonpromoter in response to c-di-GMP[J]. Nucleic acids research, 2012, 40(15): 7207-7218.</p>
          <p class="z2">[5] Baraquet C, Harwood C S. Cyclic diguanosine monophosphate represses bacterial flagella synthesisby interacting with the Walker A motif of the enhancer-binding protein FleQ[J]. Proceedings of theNational Academy of Sciences, 2013, 110(46): 18478-18483.</p>
          <p class="z2">[6] Gentner M, Allan M G, Zaehringer F, et al. Oligomer formation of the bacterial second messengerc-di-GMP: reaction rates and equilibrium constants indicate a monomeric state at physiologicalconcentrations[J]. Journal of the American Chemical Society, 2012, 134(2): 1019-1029.</p>
          <p class="z2">[7] Stamatakis M, Mantzaris N V. Comparison of deterministic and stochastic models of the lac operon	genetic network[J]. Biophysical journal, 2009, 96(3): 887-906.</p>
          <p class="z2">[8] Saltelli A, Ratto M, Andres T, et al. Global sensitivity analysis: the primer[M]. John Wiley and Sons,	2008.</p>
          <p class="z2">[9] Karzbrun E, Shin J, Bar-Ziv R H, et al. Coarse-grained dynamics of protein synthesis in a cell-free	system[J]. Physical review letters, 2011, 106(4): 048104.</p>           
        </ul>
      </hr>
        </div>
        </div>
        </div>
<body>

{% endblock %}
