{% extends "layout.html" %}

{% block page_content %}

<style>

@font-face
    {
        font-family:'font_one';
        src:url(https://static.igem.wiki/teams/4242/wiki/acme-regular.ttf)
    }
    .p1
    {
        font-family:'font_one';
        font-size:1.5rem;
        color:#f0f0f0;
    }
    
    @font-face
    {
        font-family:'font_two';
        src:url(https://uploads.igem.org/teams/4242/wiki/Overlock-Regular.ttf)

    }
    .o
    {
        font-family:'font_two';
        font-size:1.2rem;
        color:#f0f0f0;
    }
    .z1
    {
        font-family:'font_one';
        font-size:3rem;
        color:#f26b1f
    }
    .z2
    {
        font-family:'font_two';
        font-size:0.7rem;
        color:#f0f0f0
    }
    .z22
    {
        font-family:'font_two';
        font-size:0.2rem;
        color:#f0f0f0
    }
    .d1
    {
      font-family:'font_two';
        font-size:1.2rem;
        color:#f26b1f
    }
    .z4
    {
        font-family:'font_two';
        font-size:1.2rem;
        color:#f26b1f
    }
    body
   {
        width:100%;
        height: auto;
        background-color: #21373d;
        overflow-y: scroll;
    }
    #move20
    {
      margin-left: 20%;
    }

    @media screen and (max-width: 992px) 
    {
      #move20 {
      margin-left: 0%;
    }
    }
</style>


<head>
    <meta charset="UTF-8" />
    <title>Team:CUG-China/Notebook - 2022.igem.org</title>
    <meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0,user-scalable=no">
    <link rel="stylesheet" href="{{ url_for('static', filename = 'headama.css') }}">
    <link rel="stylesheet" href="{{ url_for('static', filename = 'notebook amazing/nb.css') }}">
</head>

<body>
    <section class="section main-banner" id="top" data-section="Intro">
        <img src="https://static.igem.wiki/teams/4242/wiki/background/b-res.png" alt="">
        <div class="header-text">
            <div class="caption">
                <div class="wave-text">
                    <span>Results</span>
                </div>
            </div>
        </div>
    </section>

    

    <div class="container">

        {% include "sider.html" %}
          <aside class="aside">
            <br>
            <div>
              <a class="pointer" href="#Part I">Part I</a>
              <a class="pointer" href="#Part II">Part II</a>
              <a class="pointer" href="#Part III">Part III</a>
              <a class="pointer" href="#Part IV">Part IV</a>
              <a class="pointer" href="#Part V">Part V</a>
              <a class="pointer" href="#Conclusion">Conclusion</a>
              <a class="pointer" href="#Reference">Reference</a>
            </div>
          </br>
          </aside>

          <div class="row" id="move20">     
            <div class="col" id="Content">
              <!---Part I--->
               <p class="target-fix" id="Part I"></p>
               <p style="margin-top: 50px;"></p>
               <div class="p1">
              <h1 id="Part I" style="color: #f26b1f">Part I: Biobrick Construction and transformation</h1> 
              </div> 
              <div class="o">
              <hr style="border-top:5px solid gray;">
              <ul>
                  <p class="z3"> Work we have done</p>     
                  <p>We have constructed biobrick pHYD-1 (<a  href="http://parts.igem.org/Part:BBa_K4242011" class="d1">BBa_K4242011</a>) in which reporter gene <i>gfp</i> is cloned at the downstream of promoter P<sub>pel</sub> for testing the performance of promotor P<sub>pel</sub> in <i>E. coli</i> and <i>S. oneidensis</i>. The key gene parts are shown in Fig. 1a. The ligation products were transferred to <i>E. coli</i>. To select colonies with pHYD-1, colony PCR was conducted, and the result is shown in Fig. 1b. Finally, the constructs were sequenced. </p>
                    <p style="text-align: center">
                    <img src="https://static.igem.wiki/teams/4242/wiki/results/1111.png" alt="Pulpit rock" width="630" height="230"> 
                  </p>
                  <p style="text-align: center" class="z2">Fig. 1  a|Structure of pHYD-1   b| Colony PCR result of pHYD-1</p>
                  <p>We have constructed a tandem promoter P<sub>cI/pel</sub> by using overlap PCR to fuse a constitutive promoter PcI with P<sub>pel</sub>. The PCR products directly were inserted at the upstream of <i>gfp</i> with a RBS. The resultant biobrick is named as pHYD-2 (<a  href="http://parts.igem.org/Part:BBa_K4242012" class="d1">BBa_K4242012</a>) for testing the performance of tandem promotor P<sub>cI/pel</sub> in <i>E. coli</i> and <i>S. oneidensis</i>. The structure of pHYD-2 is shown in Fig 2a. The ligation products were transferred into <i>E. coli</i>. Then, we conducted colony PCR to select colonies with pHYD-1, and the result is shown in Fig. 2b. Finally, the constructs were sequenced. </p>
                  <p style="text-align: center">
                    <img src="https://static.igem.wiki/teams/4242/wiki/results/2222.png" alt="Pulpit rock" width="630" height="230"> 
                  </p>
                  <p style="text-align: center" class="z2">Fig. 2  a|Structure of pHYD-2  b|Colony RPC result of pHYD-2</p>
                  <p>To test whether the <i>fleQ</i> can repress the <i>gfp</i> transcription driven by the tandem promoter P<sub>cI/pel</sub> in <i>E. coli</i> and <i>S. oneidensis</i>, we have constructed biobrick pHYD-3 (<a  href="http://parts.igem.org/Part:BBa_K4242013" class="d1">BBa_K4242013</a>) in which the operon of <i>fleQ</i> is placed in the front of P<sub>cI/pel</sub>- <i>gfp</i>. <i>fleQ</i> was cloned from P. aeruginosa genome. The transcription of <i>fleQ</i> is promoted by a constitutive promoter PcI. Additionally, double terminators were placed after <i>fleQ</i> and reporter genes, in order to minimize the background signal caused by gene expression leakage. The key gene parts are shown in Fig 3a. The constructs were selected by colony PCR and sequenced (Fig. 3b). </p>
                  <p style="text-align: center">
                    <img src="https://static.igem.wiki/teams/4242/wiki/results/3333.png" alt="Pulpit rock" width="630" height="230"> 
                  </p>
                  <p style="text-align: center" class="z2">Fig.3 a|Structure of pHYD-3 b| Colony PCR result of pHYD-3</p>
                  <p>To decrease intracellular c-di-GMP levels in <i>E. coli</i> and <i>S. oneidensis</i> for testing the performance of c-di-GMP biosensor, we have constructed the part P<sub>tac</sub>-<i>yhjH</i> (<a  href="http://parts.igem.org/Part:BBa_K4242017" class="d1">BBa_K4242017</a>) based on the existing part (<a  href="http://parts.igem.org/Part:BBa_K4242001" class="d1">BBa_K4242001</a>). <i>yhjH</i> gene was cloned using <i>E. coli</i> genome as templates. The expression of <i>yhjH</i> is controlled by IPTG-inducible promoter P<sub>tac</sub> (Fig. 4a). By adding different concentration of IPTG to control the expression level of <i>yhjH</i>, the intracellular c-di-GMP concentrations can be controlled at different levels. Fig 4b shows the results of electrophoresis gel of pSB3C5 cut by EcoRI.</p>
                  <p style="text-align: center">
                    <img src="https://static.igem.wiki/teams/4242/wiki/results/4444.png" alt="Pulpit rock" width="630" height="255"> 
                  </p>
                  <p style="text-align: center" class="z2">Fig.4 a|Structure of pSB3C5-<i>yhjH</i> b| The result of pSB3C5-<i>yhjH</i> digested by restriction enzyme EcoR I</p>
                  </ul>
              </hr>
              </div>

              <!---Part II--->
              <p class="target-fix" id="Part II"></p>
              <p style="margin-top: 50px;"></p>
              <div class="p1">
             <h1 id="Part II" style="color: #f26b1f">Part II : Demonstration of tandem promotor and <i><i>fleQ</i></i></h1> 
             </div> 
             <div class="o">
             <hr style="border-top:5px solid gray;">
             <ul>
                <p>In this work, we describe the design of a transcription-based fluorescent biosensor that is readily adaptable for gauging c-di-GMP levels in different bacteria. The components of the reporter comprise a transcription factor <i>fleQ</i> that is originated from P. aeruginosa, a tandem promoter P<sub>cI/pel</sub> that is composed of a constitutive promoter PcI and a c-di-GMP responsive promoter P<sub>pel</sub>, as well as a fluorescent protein <i>gfp</i> as a reporter. In P. aeruginosa, the transcriptional factor <i>fleQ</i> represses expression of the pel operon for Pel exopolysaccharide biosynthesis by a simple roadblock mechanism in which <i>fleQ</i> sits at the pel promoter and prevents RNA polymerase from binding, while <i>fleQ</i> derepresses gene expression due to the conformational change of <i>fleQ</i> upon binding c-di-GMP<sup></sup>[1]</sup>. Theoretically, the gene under pel promoter can be constitutively expressed in the bacteria that have no repressor <i>fleQ</i><sup>[2]</sup>. To test this hypothesis, we directly fused pel promoter to <i>gfp</i> and transferred the construct into two module organisms, <i>E. coli</i> BL21 and <i>S. oneidensis</i> MR-1. However, MR-1/pHYD-1 and BL21/pHYD-1 produced very low fluorescence, which suggests the promoter P<sub>pel</sub> is non-functional in these two bacteria (Fig. 5).</a></p>
                <p style="text-align: center">
                  <img src="https://static.igem.wiki/teams/4242/wiki/results/5555.png" alt="Pulpit rock" width="413" height="225"> 
                </p>
                <p style="text-align: center" class="z2">Fig.5 P<sub>cI/pel</sub> constitutively promotes expression of <i>gfp</i>. <i>fleQ</i> represses expression level of gene under the control of P<sub>cI/pel</sub></p>
                <p>To address this challenge, we designed a tandem promoter P<sub>cI/pel</sub> in which the transcription start site of the constitutive promoter PcI is immediately followed by the <i>fleQ</i> binding site of the promoter P<sub>pel</sub>. Transcription of the reporter gene <i>gfp</i> is promoted by cI promoter in the absence of <i>fleQ</i>. As shown in Fig. 5, the result shows that MR-1/pHYD-2 and BL21/pHYD-2 produced high fluorescence, indicting the tandem promotor is able to trigger the reporter gene expression. Then we cloned the regulator <i>fleQ</i> into the system. The <i>fleQ</i> successfully binds to the box and reduced the <i>gfp</i> expression in MR-1/pHYD-3 and BL21/pHYD-3 as we expected, indicating the whole system is functional in chassis like <i>E. coli</i> and <i>S. oneidensis</i> (Fig. 5).  We summarized the construction process and results in Fig. 6. For more information and details about this engineering cycles, please check <a href="https://2022.igem.wiki/cug-china/engineering" class="z4">https://2022.igem.wiki/cug-china/engineering</a></p>
                <p style="text-align: center">
                  <img src="https://static.igem.wiki/teams/4242/wiki/results/6666.png" alt="Pulpit rock" width="590" height="466">  
                </p>
                <p style="text-align: center" class="z2">Fig.6 a| Constructed vectors b| Fluorescence intensity in <i>E. coli</i> BL21 
                  c| Fluorescence intensity in <i>S. oneidensis</i> MR-1</p>
              </ul>
            </hr>
            </div>

            <!---Part III--->
            <p class="target-fix" id="Part III"></p>
            <p style="margin-top: 50px;"></p>
            <div class="p1">
           <h1 id="Part III" style="color: #f26b1f">Part III : Demonstration of c-di-GMP responding to c-di-GMP</h1> 
           </div> 
           <div class="o">
           <hr style="border-top:5px solid gray;">
           <ul>
              <p>The native c-di-GMP can bind with <i>fleQ</i>, making biosensor produce a background fluorescence. When c-di-GMP decreases, <i>fleQ</i> further represses the <i>gfp</i> expression to decrease fluorescent outputs. To demonstrate that our biosensor can respond to different intracellular c-di-GMP concentration and indicate its decrease, we decided to achieve the decrease gradient of intracellular c-di-GMP. To achieve it, we improved an existing part (<a  href="http://parts.igem.org/Part:BBa_K4242001" class="d1">BBa_K4242001</a>) encoding the <i>yhjH</i> protein, a c-di-GMP hydrolase. By changing the promotor from T7 to P<sub>tac</sub>, we can use IPTG to control the expression level of YjhH, reducing c-di-GMP concentration at different level (Fig. 7). We transformed this biobrick into the BL21/pHYD-3 and MR-1/pHYD-3, obtaining the strains BL21/ (pHYD-3+pSB3C5-<i>yhjH</i>) and MR-1/(pHYD-3+pSB3C5-<i>yhjH</i>), as well as their responding control strain BL21/(pHYD-3+pSB3C5-P<sub>tac</sub>) and MR-1/(pHYD-3+pSB3C5-P<sub>tac</sub>) (Fig. 7).</p>
              <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/results/7777.png" alt="Pulpit rock" width="630" height="230"> 
              </p>
              <p style="text-align: center" class="z2">Fig.7 Structure of the pSB3C5-P<sub>tac</sub> and pSB3C5-<i>yhjH</i></p>
              <p>We cultured <i>E. coli</i> with the addition of different concentrations of IPTG. Compared with the control strains, we observed a significant decrease in fluorescent signals when the IPTG concentration is above 0.01 mM after 12 hours (Fig. 8). This result shows that, the fluorescence intensity of strains with pHYD-3 and pSB3C5-<i>yhjH</i> decreased with the increase in the concentrations of IPTG, whereas the fluorescence intensities of the control strains, i.e., BL21/(pHYD-3+pSB3C5-P<sub>tac</sub>), had no change with the increase in IPTG concentration. </p>
              <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/results/8888.png" alt="Pulpit rock" width="538" height="400"> 
              </p>
              <p style="text-align: center" class="z2">Fig.8 Fluorescence in BL21/pHYD-3 in different c-di-GMP levels.</p>
              <p>We used MR-1/(pHYD-3+pSB3C5-YjhH) and MR-1/(pHYD-3+pSB3C5-P<sub>tac</sub>) to conduct similar experiments for testing the performance of c-di-GMP biosensor in MR-1. As shown in Fig. 9, there was no difference between the fluorescent signal of cultures without IPTG and that with 0.01 mM IPTG. However, compared to them, the fluorescent signals were significantly reduced when the concentrations of IPTG is above 0.1 mM. Meanwhile, the fluorescent signals of MR-1/(pHYD-3+pSB3C5-P<sub>tac</sub>) had no significant difference. Collectively, our results showed that IPTG had no impact on the production of fluorescent proteins, and our biosensor can convert the decrease in c-di-GMP levels to the decreased fluorescence which is easily detected.</p>
              <p style="text-align: center">
                <img src="https://static.igem.wiki/teams/4242/wiki/results/9999.png" alt="Pulpit rock" width="532" height="400"> 
              </p>
              <p style="text-align: center" class="z2">Fig.9 Fluorescence intensity of MR-1/pHYD-3 with different c-di-GMP levels.</p>
            </ul>
          </hr>
          </div>

          <!---Part IV--->
          <p class="target-fix" id="Part IV"></p>
          <p style="margin-top: 50px;"></p>
          <div class="p1">
         <h1 id="Part IV" style="color: #f26b1f">Part IV : Verification of biosensor screening biofilm dispersal agents</h1> 
         </div> 
         <div class="o">
         <hr style="border-top:5px solid gray;">
         <ul>
            <p>A group of structurally diverse compounds are known to hinder the formation of bacterial biofilm or trigger the dispersal of biofilm.<sup>[3]</sup> It is also known that biofilm dispersal can be induced by sequestering cellular c-di-GMP, which indicates that a low c-di-GMP level could directly lead to biofilm dispersal. By using the biosensor, we thought whether some of the biofilm-dispersing compounds that cause a reduction in cellular c-di-GMP levels can be selected in <i>E. coli</i> and <i>S. oneidensis</i>. The answer to the question would determine whether we can apply our biosensor into the development of new chemicals or technical to repress the biofilm formation. Based on the reported study, the compounds including D-Tyrosine, 3-Iodolylacetonitrile and Resveratrol are used in our projects (Fig. 10). The concentrations of the biofilm-dispersing compounds used in this project are listed in Table S1<sup>[3]</sup> in the page (<a href="https://2022.igem.wiki/cug-china/proof of concept" class="z4">https://2022.igem.wiki/cug-china/proof of concept</a>). It was reported that these compounds are able to disperse biofilm in <i>E. coli</i> and other bacterial strains. The measurement of the fluorescence intensity were performed following the protocol in the protocol page<a href="https://2022.igem.wiki/cug-china/protocol" class="z4">https://2022.igem.wiki/cug-china/protocol</a></p>
            <p style="text-align: center">
              <img src="https://static.igem.wiki/teams/4242/wiki/results/0000.png" alt="Pulpit rock" width="638" height="320"> 
            </p>
            <p style="text-align: center" class="z2">Fig.10 Biofilm-dispersing compounds used in our experiments.</p>
            <p>The strains BL21/pHYD-3 and MR-1/pHYD-3 were treated with biofilm-dispersing agents mentioned above. The individual compounds were added separately to the medium at the beginning of the cell culture. Then they were incubated at 37℃ and 30℃ for 12 hours, respectively. Then we harvested the cells and measured fluorescence by the protocol. As shown in Fig. 11, a decrease of the fluorescence intensity was observed for all three compounds in BL21 strains. Compared with the control group, i.e., BL21/pHYD-3 without the treatment of biofilm-dispersing agents, these three compounds respectively reduced fluorescence intensity about 23%, 27% and 13%. The results are consistent with the reported study.</p>
            <p style="text-align: center">
              <img src="https://static.igem.wiki/teams/4242/wiki/results/01010.png" alt="Pulpit rock" width="480" height="400"> 
            </p>
            <p style="text-align: center" class="z2">Fig.11 Fluorescence intensity of MR-1/pHYD-3, cultured with different dispersal agents. Two-sided Student's t test was used to analyze the statistical significance (*P&lt; 0.05)</p>
            <p>Similarly, we treated the strain MR-1/pHYD-3 with these biofilm-dispersing agents. Compared to the control group in which MR-1/pHYD-3 cells were grown without biofilm-dispersing agents, the treatments of biofilm-dispersing agents significantly reduced the fluorescence of MR-1/pHYD-3 that containing our c-di-GMP biosensor (Fig. 12). The results suggested that these compounds could reduce intracellular c-di-GMP level of MR-1. </p>
            <p style="text-align: center">
              <img src="https://static.igem.wiki/teams/4242/wiki/results/1313.png" alt="Pulpit rock" width="522" height="400"> 
            </p>
            <p style="text-align: center" class="z2">Fig.12 Fluorescence intensity of MR-1/pHYD-3, cultured with different dispersal agents. Two-sided Student's t test was used to analyze the statistical significance (*P&lt; 0.05)</p>
            <p>To verify whether these compounds indeed reduced the intracellular c-di-GMP of MR-1, we cultured the strains again under the same condition and harvest the cells to measure its intracellular c-di-GMP concentration. On the basis of the curves measured by HPLC, we observed that the compounds caused a drop of intracellular c-di-GMP concentration from 1.71 pmol/μg protein to 0.44 pmol/μg total protein (Fig. 13), which was reduced by about 73%. These results suggested that our biosensor is able to monitor these compounds inducing biofilm dispersal by reducing intracellular c-di-GMP level.</p>
            <p style="text-align: center">
              <img src="https://static.igem.wiki/teams/4242/wiki/results/1414.png" alt="Pulpit rock" width="522" height="400"> 
            </p>
            <p style="text-align: center" class="z2">Fig.13  C-di-GMP level in MR-1/pHYD-3 measured by HPLC.</p>
            <p>The biosensors revealed a decline of c-di-GMP level when the cells were treated with biofilm-dispersing agents. The result establishes the biosensor as valuable tool for using in chemical biology and biofilm-dispersal agent screening. However, analyzing the data of our experiments also exhibit some problems in our biosensor. As shown in Fig.11, 12 and 13, when treated with biofilm dispersal agents for 6 hours, the reduction of fluorescence varies between 10%-27%, which is out of proportion with the decrease of c-di-GMP level (about 73%). It means the dynamic range of the detection is not wide enough. Although it can indicate the drop of c-di-GMP level, it can't precisely reflect the dynamic change of c-di-GMP due to its narrow dynamic range,</p>
            
          </ul>
        </hr>
        </div>

        <!---Part V--->
        <p class="target-fix" id="Part V"></p>
        <p style="margin-top: 50px;"></p>
        <div class="p1">
       <h1 id="Part V" style="color: #f26b1f">Part V Work has not finished yet</h1> 
       </div> 
       <div class="o">
       <hr style="border-top:5px solid gray;">
       <ul>
          <p>Based on our current data, we found that our biosensor exhibits a narrow dynamic range. One of factor for the narrow dynamic range could because of the low native c-di-GMP level or the high expression level of <i>fleQ</i>. We cannot change the native c-di-GMP level, but can control the expression level of <i>fleQ</i> by using inducible promoter. Thus, we will preliminary improve the dynamic range of our biosensor by controlling the expression of regulator <i>fleQ</i> with a arabinose inducible promoter PBAD. The biobrick design is shown in Fig. 14a. The constitutive promoter PcI in pHYD-3 is replaced by the inducible promoter PBAD, and the resultant biobrick is named pHYD-B1. </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/results/npcsb.png" alt="Pulpit rock" width="690" height="320"> 
          </p>
          <p style="text-align: center" class="z2">Fig.14 a|improvement of pHYD-3 by changing promotor. b|Colony RPC result of pHYD-B1</p>
          <p>It has been reported that <i>fleQ</i> can act as the activator of P<sub>pel</sub> in the presence of another regulator <i>fleN</i>. To further widen the dynamic range of this c-di-GMP biosensor, we cloned regulator gene <i>fleN</i> into pHYD-B1, resulting in the biobrick pHYD-B2 (Fig. 15).  Although we have constructed these biobricks (<a  href="http://parts.igem.org/Part:BBa_K4242014" class="d1">BBa_K4242014</a>, <a  href="http://parts.igem.org/Part:BBa_K4242015" class="d1">BBa_K4242015</a>) and added these parts in the registry, their demonstration have note done because of the limited time. </p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/results/jzl.png" alt="Pulpit rock" width="690" height="320"> 
          </p>
          <p style="text-align: center" class="z2">Fig.15 a|improvement of pHYD-B1 by adding another regulator <i>fleN</i>. b|Colony RPC result of pHYD-B2</p>
          <p>We also inserted pHYD-B1 into the <i>E. coli</i> BL21 and measured its growth curves under different arabinose concentrations (Fig. 16). However, we don't have enough time to finish further experiments for their demonstration. The results showed that the increase in arabinose concentrations slightly reduced the growth rate of <i>E. coli</i> BL21.</p>
          <p style="text-align: center">
            <img src="https://static.igem.wiki/teams/4242/wiki/results/zfx.png" alt="Pulpit rock" width="522" height="400"> 
          </p>
          <p style="text-align: center" class="z2">Fig.16 Growth curves of <i>E. coli</i> BL21/pHYD-B1 under different Arabinose concentrations.</p>
          <p>We have constructed the gene part pSB3C5-<i>yhjH</i> to decrease intracellular c-di-GMP concentration at different levels for testing the performance of c-di-GMP biosensor, simulating the effect of biofilm-dispersing agents that can induce biofilm dispersal by decreasing c-di-GMP levels. In addition to “bad” biofilms, there are a lot of “good” biofilms which have a wide range of biotechnology applications. For example, <i>S. oneidensis</i> is a model electroactive bacterium that can exchange electron with electrodes in bioelectrochemical systems (BESs). Specially, it shows promise as biocatalysts for efficient conversion of a wide range of organic wastes and renewable biomass to electricity. This application is greatly associated with the formation of <i>S. oneidensis</i> biofilms on the anode. Increasing c-di-GMP level to enhance anode biofilms of <i>S. oneidensis</i> is a promising strategy to promote electron transfer and increase electricity generation. Our biosensors can be used to screening the c-di-GMP synthase with high enzyme activity. Therefore, we need to check whether our biosensor can detect the increase in c-di-GMP levels. To increase intracellular c-di-GMP level, we cloned a c-di-GMP synthase gene, yedQ. We have finished the construction of this part (<a  href="http://parts.igem.org/Part:BBa_K4242016" class="d1">BBa_K4242016</a>) and plan to finish its demonstration in next year.</p>
          
      </ul>
      </hr>
      </div>

      <!---Conclusion--->
      <p class="target-fix" id="Conclusion"></p>
      <p style="margin-top: 50px;"></p>
      <div class="p1">
        <h1 id="Conclusion" style="color: #f26b1f">Conclusion</h1> 
        </div> 
        <div class="o">
        <hr style="border-top:5px solid gray;">
        <ul>
      <p>Collectively, we can see a clear path of the engineering cycles and construction of the biosensor, testing its reaction to both endogeneity and exogenous decrease of c-di-GMP level. The biosensor also revealed a decline of c-di-GMP level wen the cells were treated with biofilm dispersing agents. Now we can come to conclusion that we have successfully developed a broadly applicable biosensor for gauging intracellular c-di-GMP levels, which can be used to screening biofilm dispersal agents and other biofilm/c-di-GMP associated research. By studying the biosensor-containing cells embedded in biofilm matrixes, we will also be able to gain a better understanding of the roles c-di-GMP played in the highly dynamic and complex processes of biofilm formation and dispersal. In future, we will continue to widen the dynamic range of our biosensor and explore its wider applications, which matches the goal of applying synthetic biology to make the world better.</p>
    </ul>
    </hr>
    </div>

    <!---Reference--->
    <p class="target-fix" id="Reference"></p>
    <p style="margin-top: 50px;"></p>
    <div class="p1">
   <h1 id="Reference" style="color: #f26b1f">Reference</h1> 
   </div> 
   <div class="o">
   <hr style="border-top:5px solid gray;">
   <ul>
    <p class="z2">[1]	Baraquet C, Murakami K, Parsek M R, et al. The FleQ protein from Pseudomonas aeruginosa functions as both a repressor and an activator to control gene expression from the pel operon promoter in response to  c-di-GMP[J]. Nucleic Acids Res, 2012,40(15):7207-7218.</p>
    <p class="z2">[2]	Hickman J W, Harwood C S. Identification of FleQ from Pseudomonas aeruginosa as ac-di-GMP-responsive transcription factor[J]. Molecular microbiology, 2008,69(2):376-389.</p>
    <p class="z2">[3]	Ho C L, Chong K S J, Oppong J A, et al. Visualizing the perturbation of cellular cyclic di-GMP levels in bacterial cells[J]. Journal of the American Chemical Society, 2013,135(2):566-569.</p>
  </ul>
</div>





            
      </div>
      </div>
    </div>
</body>

{% endblock %}
